src/controllers/__tests__/services.test.ts

```diff
@@ -27,7 +28,7 @@ const organizationFiscalCode = "0123455" as OrganizationFiscalCode;
 
 const userContract: IExtendedUserContract = {
   email: "test@test.it",
-  groupNames: ["groupp_1", "group_2"] as any,
+  groupNames: new SerializableSet(["groupp_1", "group_2"]),
   id: "124123_id",
   name: "name"
 };
```

Comment:
Hi Daniele, just for my clarification, I'm wondering why init stuff like this isn't inside a "BeforeAll" Jest handler. Thanks.

Reply:
Hi @LoFrance. We prefer works with immutable constant to reduce _side effects_. If we use `beforeAll` hooks to initialize variables before tests, we need to declare it with `let` instead of `const`.

Reply:
Thanks @BurnedMarshal, so it's a functional approach design to control and reduce _side effects_. I'm not so comfortable with it and I didn't catch it, so thank you for this explanation.